src/main/java/ua/softserveinc/tc/controller/admin/DayOffController.java

```diff
@@ -64,6 +68,7 @@
         if (dayOffService.dayOffExist(dayOff.getName(), dayOff.getStartDate())) {
             log.warn("There is another day off with the same name: " + dayOff.getName() + ", or" +
                     "with the same start date: " + dayOff.getStartDate());
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
```

Comment:
"In the past method creates this dayof and only produces warning, but now it doesn.t create the dayoff" This is wrong, so I change it) The method is calling "createDayOff()", then it check if this event name and start date (I will change it to be as if this event between startDate and endDate exists) exists at all, if yes -> we don't write it to DB, but if no we'll create it. What was donw before? It creates dayOff even if this dayOff already exists.